cluster/src/main/java/org/apache/iotdb/cluster/ClusterIoTDB.java

```diff
@@ -390,12 +399,11 @@ private void startClientRPC() throws QueryProcessException, StartupException {
     // we must wait until the metaGroup established.
     // So that the ClusterRPCService can work.
     ClusterTSServiceImpl clusterServiceImpl = new ClusterTSServiceImpl();
-    clusterServiceImpl.setCoordinator(coordinator);
-    clusterServiceImpl.setExecutor(metaGroupMember);
-    BasicServiceProvider.sessionManager = ClusterSessionManager.getInstance();
+    ServiceProvider.SESSION_MANAGER = ClusterSessionManager.getInstance();
     ClusterSessionManager.getInstance().setCoordinator(coordinator);
     ClusterRPCService.getInstance().initSyncedServiceImpl(clusterServiceImpl);
     registerManager.register(ClusterRPCService.getInstance());
+    IoTDB.initInfluxDBMManager();
```

Comment:
`if (IoTDBDescriptor.getInstance().getConfig().isEnableInfluxDBRpcService()) { }`
 ?

Reply:
OK, I will add this.